assets/app/view/map_page.rb

```diff
@@ -14,7 +14,7 @@ def render
 
       return h(:p, "Bad game title: #{game_title}") unless game_class
 
-      names = %w[p1 p2 p3 p4]
+      names = %w[p1 p2 p3 p4 p5]
```

Comment:
this breaks games that don't support 5 players like 1836jr.

in lib/engine.rb i have a function called player_range